src/routes/(console)/project-[region]-[project]/overview/platforms/createReactNative.svelte

```diff
@@ -35,9 +35,8 @@
     const gitCloneCode =
         '\ngit clone https://github.com/appwrite/starter-for-react-native\ncd starter-for-react-native\n';
 
-    const updateConfigCode = `const EXPO_PUBLIC_APPWRITE_PROJECT_ID = "${projectId}";
-const EXPO_PUBLIC_APPWRITE_ENDPOINT = "${sdk.forProject(page.params.region, page.params.project).client.config.endpoint}";
-        `;
+    const updateConfigCode = () => `EXPO_PUBLIC_APPWRITE_PROJECT_ID=${projectId}
```

Comment:
```suggestion
    const updateConfigCode = `EXPO_PUBLIC_APPWRITE_PROJECT_ID=${projectId}
```
we don't need a function call as well. a `const` variable is OK.